docs/built-in-functions.rst

```diff
@@ -186,10 +186,10 @@ Returns ``sha256`` (SHA2 256bit output) hash of input.
     :param a:
     :type a: decimal, larger than 0.0
 
-    :output sqrt: decimal
+    :output decimal: decimal
```

Comment:
```suggestion
    :output b: decimal
```

Reply:
Yeah. I missed this in the last commit.

Reply:
You can commit this suggestion using the GitHub interface above if you want